### DISCUSSION TITLE
Is there a way to specify the web browser for the add to cart/open tab function? Whats the defualt?

### DISCUSSION THREAD

**Discussion by Niswendel** [jef/streetmerchant#1014] [Help]
And does the software open a new tab in say chromium or open a new version of chromium (meaning a new set of tabs?

**Comment by thefl0yd** [comment]
You've posted over half a dozen questions/comments/complaints and assigned them to someone who does this as a hobby, to help us, in their free time.

Do you think you're being realistic here? Either read up - read the wiki, parse the code, whatever else it takes and become an informed user. This is not a 'we will do it for you and explain it to you every step of the way' type of software package and the maintainer already graciously donates more of his time than most people would. Become an informed user or maybe use one of the paid services that exist to find whatever it is you're looking for.

**Comment by Niswendel** [comment]
Thats fair, ill make this my last question?

**Comment by Niswendel** [comment]
sorry ment to end in a period.

**Comment by jef** [comment]
No worries, @Niswendel. Thanks @thefl0yd 😅 

![image](https://user-images.githubusercontent.com/12074633/101531616-828fb880-3961-11eb-9a8e-49e305f30785.png)

It opens a new tab if available. Otherwise, new browser.

https://jef.codes/streetmerchant/reference/application/

**Reply by Kutalia** [reply]
Thank you for taking your time for developing such an useful software.
I think the question was the following: in which browser will the bot add items to a cart?
I know it's usually default browser when it's in `headless` mode, but what happens when `headless` is disabled? Will it open a new Chromium tab if both Google Chrome (default browser) and Chromium are open?
